tests/integration/components/paper-form-field-test.js

```diff
@@ -198,4 +198,42 @@ module('Integration | Component | {{paper-form-field}}', function(hooks) {
     assert.equal(this.get('object.name'), 'John White',
                  `doesn't update object value`);
   });
+
+   test(`Shows a after content depending on the given content`, async function(assert) {
+    this.set('object', EmberObject.create({
+      text: 'voorbeeld'
+    }));
+
+    await render(hbs`
+      {{paper-form-field type="text-field"
+                         object=object
+                         after="after tekst"
+                         propertyPath="text"
+                         updated=(action 'update')}}`);
+
+    let $input = this.element.querySelector('input.md-input').value;
+    assert.equal($input, 'voorbeeld', 'shown component has correct value');
+
+    let $component = this.element.querySelector('.form-field--has-after');
```

Comment:
`this.element.querySelector('.form-field');`, you want to check if the `.form-field` has a class `.form-field--has-class'.